test/unit/foreman/renderer/template_files.yaml

```diff
@@ -0,0 +1,67 @@
+files:
```

Comment:
A comment "Edit this file and then run rake snapshot..." would be useful here, just a nitpick if you intent to rebase.